WcaOnRails/app/webpacker/components/Delegates/index.jsx

```diff
@@ -35,12 +36,28 @@ export default function Delegates() {
     [delegateGroups],
   );
 
-  const [activeRegion, setActiveRegion] = React.useState();
-  const [adminMode, setAdminMode] = React.useState(false);
+  const [hash, setHash] = useHash();
+
+  const SelectedComponent = React.useMemo(() => {
+    const selectedRegionIndex = delegateRegions.findIndex((region) => region.id === +hash);
+    if (selectedRegionIndex === -1) {
+      setHash(delegateRegions[0]?.id);
```

Comment:
Will this work if `delegateRegions` is empty? Like while the page is loading, `delegateRegions` will be empty right?

Reply:
If `delegateRegions` is empty, it will show the `Loading` component, I think it is expected behavior